src/main/java/net/vulkanmod/mixin/window/WindowMixin.java

```diff
@@ -70,6 +70,8 @@ private GLCapabilities redirect2() {
     @Inject(method = "<init>", at = @At(value = "INVOKE", target = "Lorg/lwjgl/glfw/GLFW;glfwCreateWindow(IILjava/lang/CharSequence;JJ)J"))
     private void vulkanHint(WindowEventHandler windowEventHandler, ScreenManager screenManager, DisplayData displayData, String string, String string2, CallbackInfo ci) {
         GLFW.glfwWindowHint(GLFW_CLIENT_API, GLFW_NO_API);
+        //Fix Gnome Client-Side Decorators
+        GLFW.glfwWindowHint(GLFW_DECORATED, VideoResolution.GNOME() && VideoResolution.isWayLand() ? GLFW_FALSE : GLFW_TRUE);
```

Comment:
I'm not too sure about this approach honestly. Better just keep `isWayland`. Right now I'm having the issue in KDE, maybe someone will have it in Hyprland or some other compositor. Maybe ditch the code for checking for the DE in general and just allow it in case it's running under wayland?